meson.build

```diff
@@ -268,6 +268,11 @@ if get_option('tests') != 'false'
         endif
 endif
 
+ossfuzz = get_option('oss-fuzz')
+if ossfuzz
+        fuzzing_engine = meson.get_compiler('cpp').find_library('FuzzingEngine')
```

Comment:
I think it would make sense to search for `Fuzzer` if `FuzzingEngine` is not found. This should simplify building the fuzzers a little.

Reply:
Done.